c7n/resources/kms.py

```diff
@@ -3,6 +3,7 @@
 from botocore.exceptions import ClientError
 
 import json
+from functools import cached_property
```

Comment:
i was curious about this ... and discovered it's only in 3.8+ :|

Reply:
Womp womp... :)